crates/core/src/factories/core_fetch/back_off.rs

```diff
@@ -8,9 +8,9 @@ use kitsune2_api::AgentId;
 
 #[derive(Debug)]
 pub struct BackOffList {
-    state: HashMap<AgentId, BackOff>,
-    first_back_off_interval: Duration,
-    last_back_off_interval: Duration,
+    pub state: HashMap<AgentId, BackOff>,
```

Comment:
Is this really `pub` or `pub(crate)`?

Reply:
I think it makes no difference because the module `back_off` is not `pub`.

Reply:
I guess so yes, I just prefer to be as restrictive as possible :)

Reply:
Okay, restrictive works for me.

Reply:
Restricted the hell out of it.